server/settings/creator_plugins.py

```diff
@@ -231,13 +238,23 @@ class TrayPublisherCreatePluginsModel(BaseSettingsModel):
         title="Batch Movie Creator",
         default_factory=BatchMovieCreatorPlugin
     )
+    editorial_creators: TraypublisherEditorialCreatorPlugins = SettingsField(
+        title="Editorial Creators",
+        default_factory=TraypublisherEditorialCreatorPlugins,
+    )
     IngestCSV: IngestCSVPluginModel = SettingsField(
         title="Ingest CSV",
         default_factory=IngestCSVPluginModel
     )
+    simple_creators: list[SimpleCreatorPlugin] = SettingsField(
```

Comment:
If we'll move it here, it should be at first place.

Reply:
I'll make it first  https://github.com/ynput/ayon-traypublisher/pull/44/commits/b7dbe8b6f9534cda27e00d39d98bab228e3c3b4b

and we can decide if we want to keep them or revert the changes.